app/views/journals/show.html.erb

```diff
@@ -24,3 +24,4 @@
     <td><%= @journal.goals_tomorrow %></td>
   </tr>
 </table>
+<%= link_to t('buttons.download'), download_user_journal_path, target: '_blank' %>
```

Comment:
Evea suggested me to use a button here, i guess the same applies to you

Reply:
I tried to use `button_to`, but the user experience suffers because I can't get the button to open a new tab where you can view the PDF and download it. I can adjust the `link_to` with some SCSS styling so that it looks like a button if that is acceptable.

Reply:
Sure you can do it with CSS